windows/LICENSE.rtf

```diff
@@ -0,0 +1,206 @@
+{\rtf1\ansi\ansicpg1252\deff0\nouicompat\deflang1033{\fonttbl{\f0\fnil\fcharset0 Tahoma;}}
```

Comment:
What's the purpose of a rich text version of the license if we copy the normal version during the release flow? I'm a bit uneasy about having multiple copies of the license in the repo. If we ever change the license we'll need to know this is here.

Reply:
We have a screen like this:
![Screen Shot 2022-03-27 at 8 45 04 PM](https://user-images.githubusercontent.com/10042942/160309053-6a0727a7-b2cf-46b5-aadb-043a0c751f96.png)

If we don't format it, it looks really bad and janky. So we basically have 2 options: 

1. remove this license agreement step and just have the agreement be implicit by using our software. This is mostly what we were already doing before, and basically what we're doing on linux.
2. Add an rtf formatted license for this screen that makes the screen look correct. This is the option in this PR right now.

I figured no. 2 was preferred, since we're bundling the license and it makes sense to have the customer agree to the license before installing. It seems like the "right" way to handle it.

Reply:
ah gotcha. Ok I'm fine with that.